Directory.Build.props

```diff
@@ -1,27 +1,43 @@
 <Project>
-	<!--Nightly NuGets only!!!!!!!!!!!-->
-	<PropertyGroup>
-		<!--For use with Nightly packages-->
-		<!--REMINDER: When packing, use the proper ISO layout, i.e ddHH-->
-		<!--<CurrentDate>$([System.DateTime]::Now.ToString(ddHH))</CurrentDate>-->
+	<Choose>
+		<When Condition="'$(Configuration)' == 'Installer'">
+			<PropertyGroup>
+				<Minor>$([System.DateTime]::Now.ToString(MM))</Minor>
+				<Build>$([System.DateTime]::Now.get_DayOfYear().ToString())</Build>
 
-		<Minor>$([System.DateTime]::Now.ToString(yyMM))</Minor>
-		<Build>$([System.DateTime]::Now.get_DayOfYear().ToString())</Build>
+				<!--As WIX installer can only cope with build numbers <= 256, the yyMM layout won't work. 
```

Comment:
This is not going to work, as 2212 will be the same as 2112

Reply:
Minor = yy
Release = MM
Build = doy

Or Minor = yy
Build = doy.

Reply:
Or the revision becomes doy i.e 6.22.01.1?

Reply:
> Or the revision becomes doy i.e 6.22.01.1?

Yes, use all 4 number slots :-)